src/metabase/query_processor/store.clj

```diff
@@ -241,14 +241,14 @@
 (def ^:private ^{:deprecated "0.48.0"} LegacyFieldMetadata
   [:map
    [:name          ms/NonBlankString]
-   [:table_id      ::lib.schema.common/positive-int]
+   [:table_id      ::lib.schema.id/table]
```

Comment:
preferrable to be more specific, more than once I've found myself unconfused by what an `:id` key was supposed to be an ID for by looking at the schema. One day this stuff will come in handy when we do generative nonsense